### PR TITLE
[No QA] Fix various propType warnings

### PR DESCRIPTION
**Pull request by marcaaron**
### Fixed Issues
Fixes No Issue just many propTypes warnings making it hard to debug mobile

### Tests / QA Steps
No QA

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [x] iOS
- [ ] Android

### CODE REVIEW

**Review by TomatoToaster — APPROVED** [review]

**Comment by TomatoToaster** [comment]
I think this looks good, just asked @Jag96 to double check me though.

File: src/pages/home/sidebar/OptionRow.js [hunk 1]
```diff
@@ -130,12 +130,12 @@ const OptionRow = ({
...
                                         optionIsFocused={optionIsFocused}
                                         size={mode === 'compact' ? 'small' : 'default'}
-                                        styles={hovered && !optionIsFocused && {
+                                        styles={(hovered && !optionIsFocused) ? {
                                             secondAvatar: {
                                                 backgroundColor: themeColors.sidebarHover,
                                                 borderColor: themeColors.sidebarHover,
                                             },
-                                        }}
+                                        } : undefined}
```
Comment (Jag96) [inline src/pages/home/sidebar/OptionRow.js:138]: Just wondering, what was the warning here? Having `undefined` here seems less clean

**Review by Jag96 — COMMENTED** [review]
Just one question, otherwise LGTM

File: src/pages/home/sidebar/OptionRow.js (hunk 1 above)
Comment (marcaaron) [reply src/pages/home/sidebar/OptionRow.js:138]: also asked Rajat to fix this up in another PR

**Review by Jag96 — APPROVED** [review]

**Pull request merged**